### PR TITLE
Added documentation missing entry for error 12009

### PR DESCRIPTION
**Pull request by danielgwilson**
Added documentation missing entry for error 12009 in Errors from displayDialogAsync section. This will resolve bug 2544119 in DevOps.

### CODE REVIEW

File: docs/develop/dialog-api-in-office-add-ins.md [hunk 1]
```diff
@@ -247,6 +247,7 @@ In addition to general platform and system errors, three errors are specific to
 |12004|The domain of the URL passed to `displayDialogAsync` is not trusted. The domain must be the same domain as the host page (including protocol and port number).|
 |12005|The URL passed to `displayDialogAsync` uses the HTTP protocol. HTTPS is required. (In some versions of Office, the error message returned with 12005 is the same one returned for 12004.)|
 |<span id="12007">12007</span>|A dialog box is already opened from this host window. A host window, such as a task pane, can only have one dialog box open at a time.|
+|12009|The user chose to ignore the dialog box. This can occur in the online versions of Office where users are given the option to not allow an Add-in to present a dialog.|
```
Comment (AlexJerabek) [inline docs/develop/dialog-api-in-office-add-ins.md]: Couple nits: "add-in" shouldn't be capitalized here. We could also condense that second sentence a bit. How does the following sound?
"This occurs in online versions of Office when users do not allow an add-in to present a dialog."

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/dialog-api-in-office-add-ins.md](https://github.com/danielgwilson/office-js-docs-pr/blob/patch-1/docs/develop/dialog-api-in-office-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C22%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810222134098873-501%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=smqPwVAM9LG5Q%2B9K1%2BozqXeaFczYCP27nkSjUp3Ih1g%3D&st=2018-10-22T21%3A30%3A33Z&se=2018-11-22T21%3A35%3A33Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/develop/dialog-api-in-office-add-ins.md (hunk 1 above)
Comment (Rick-Kirkham) [inline docs/develop/dialog-api-in-office-add-ins.md]: I think the first sentence should be "The user has blocked dialogs in add-ins."

**Comment by Rick-Kirkham** [comment]
This article also needs to be changed. Please add the new 12009 to the table that current ends with 12007. It's a bit more than halfway down the page.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/dialog-api-in-office-add-ins.md](https://github.com/danielgwilson/office-js-docs-pr/blob/patch-1/docs/develop/dialog-api-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/reference/javascript-api-for-office-error-codes.md](https://github.com/danielgwilson/office-js-docs-pr/blob/patch-1/docs/reference/javascript-api-for-office-error-codes.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C31%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810311911009681-501%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=p0WoM850lXA5r1EjYBX%2BN6Vikc4Rx5x67Z03XkihhQg%3D&st=2018-10-31T19%3A07%3A20Z&se=2018-12-01T19%3A12%3A20Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/develop/dialog-api-in-office-add-ins.md (hunk 1 above)
Comment (kbrandl) [reply docs/develop/dialog-api-in-office-add-ins.md]: I've updated the second sentence; resolving this conversation.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/dialog-api-in-office-add-ins.md](https://github.com/danielgwilson/office-js-docs-pr/blob/patch-1/docs/develop/dialog-api-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/reference/javascript-api-for-office-error-codes.md](https://github.com/danielgwilson/office-js-docs-pr/blob/patch-1/docs/reference/javascript-api-for-office-error-codes.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C31%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810311920480625-501%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=eITKH6LZlhYOHTHdyCLSzWifsEooDejprBaGWqsB07Y%3D&st=2018-10-31T19%3A17%3A04Z&se=2018-12-01T19%3A22%3A04Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**